### PR TITLE
Simplify the travis.yml file

### PR DESCRIPTION
**Pull request by aaronbrethorst**
I'm pretty sure we can get away with not having a bunch of the stuff that was in our travis.yml file in our travis.yml file.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,18 +1,11 @@
-osx_image: xcode7.1
+osx_image: xcode7.2
 language: objective-c
-before_install:
-  - brew update
-  - brew outdated xctool || brew upgrade xctool
```
Comment (bbodenmiller) [inline .travis.yml]: This was just to make sure the latest version of `xctool` was installed. Before the Travis images weren't keeping up with the updates very frequently.

File: .travis.yml [hunk 2]
```diff
@@ -1,18 +1,11 @@
...
-  - gem install cocoapods
-  - pod install
```
Comment (bbodenmiller) [inline .travis.yml]: I think you need this to install the cocoapods?

File: .travis.yml (hunk 1 above)
Comment (aaronbrethorst) [inline .travis.yml]: ahh gotcha, thanks for the info.

File: .travis.yml (hunk 2 above)
Comment (bbodenmiller) [inline .travis.yml]: Actually looks like this may run automatically now: https://docs.travis-ci.com/user/languages/objective-c/#Dependency-Management

Comment (aaronbrethorst) [inline .travis.yml]: the docs seem to imply that they do this implicitly if you have a Podfile checked in. I thought I'd test it in the best way I know how ;-)

**Pull request closed without merging**